utils/fixed_decimal/src/decimal.rs

```diff
@@ -989,6 +1008,35 @@ impl FixedDecimal {
     /// assert_eq!("1", dec.to_string());
     /// ```
     pub fn trunc(&mut self, position: i16) {
+        self.trunc_coarse(position, RoundingCoarseness::MultiplesOf1)
+    }
+
+    /// Truncates the number on the right to a particular position and rounding
+    /// coarseness, deleting digits if necessary.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use fixed_decimal::{FixedDecimal, RoundingCoarseness};
+    /// # use std::str::FromStr;
+    ///
+    /// let mut dec = FixedDecimal::from_str("-3.5").unwrap();
+    /// dec.trunc_coarse(0, RoundingCoarseness::MultiplesOf2);
+    /// assert_eq!("-2", dec.to_string());
+    /// let mut dec = FixedDecimal::from_str("7.57").unwrap();
+    /// dec.trunc_coarse(-1, RoundingCoarseness::MultiplesOf5);
+    /// assert_eq!("7.5", dec.to_string());
+    /// let mut dec = FixedDecimal::from_str("5.45").unwrap();
+    /// dec.trunc_coarse(-2, RoundingCoarseness::MultiplesOf25);
+    /// assert_eq!("5.25", dec.to_string());
+    /// let mut dec = FixedDecimal::from_str("9.99").unwrap();
+    /// dec.trunc_coarse(-1, RoundingCoarseness::MultiplesOf25);
+    /// assert_eq!("7.5", dec.to_string());
+    /// let mut dec = FixedDecimal::from_str("9.99").unwrap();
+    /// dec.trunc_coarse(-2, RoundingCoarseness::MultiplesOf2);
+    /// assert_eq!("9.98", dec.to_string());
+    /// ```
+    pub fn trunc_coarse(&mut self, position: i16, coarseness: RoundingCoarseness) {
```

Comment:
naming: using "truncation" and "rounding" in the same API is confusing. This API never rounds, so `RoundingCoarseness` should probably be `TruncationCoarseness`, or some neutral term.

Reply:
Though, ECMA402 considers truncation as a "roundingMode"...
The obvious problem would be that `TruncationCoarseness` has the same problem when used in `expand_coarse`, since the API never truncates, and I can't find another term to encompass all operations.

Maybe we can just leave it as `Coarseness`?